bedrock/firefox/templates/firefox/new/desktop/thanks.html

```diff
@@ -80,6 +80,14 @@ <h1 class="c-section-title show-unknown">{{ ftl('firefox-desktop-download-interr
         {% set support_windows_attrs = 'href="https://support.mozilla.org/kb/how-download-and-install-firefox-windows%s" rel="external noopener" data-cta-type="link" data-cta-text="Get help with your installation"'|safe|format(referrals) %}
         {{ ftl('firefox-desktop-download-get-help', attrs=support_windows_attrs) }}
       </p>
+      <div class="show-linux">
```

Comment:
Might it be worth using a different title here for Linux? The title here kind of implies that the download has already happened. Maybe something like "Almost there! Select a download option below."?

![image](https://github.com/mozilla/bedrock/assets/400117/4fb966f0-2ecc-4b51-a83d-4e9aff85d3db)